build.gradle.kts

```diff
@@ -1,16 +1,13 @@
-import net.researchgate.release.GitAdapter.GitConfig
-import net.researchgate.release.ReleaseExtension
-
 description = "An application to reset the state of Kafka Connect connectors"
 
 plugins {
     `java-library`
-    id("net.researchgate.release") version "2.8.1"
+    id("net.researchgate.release") version "3.0.2"
     id("com.bakdata.sonar") version "1.1.11"
     id("com.bakdata.sonatype") version "1.1.11"
-    id("org.hildan.github.changelog") version "0.8.0"
-    id("com.google.cloud.tools.jib") version "3.3.0"
-    id("io.freefair.lombok") version "5.3.3.3"
+    id("org.hildan.github.changelog") version "1.12.1"
```

Comment:
From my experience, the upgrade to `2.0.0` went smoothly in other projects.
If you wish to go for a non-major update then let stick with [1.13.1](https://github.com/joffrey-bion/gradle-github-changelog/releases/tag/1.13.1)